src/components/SocialLocationPreview/SocialLocationPreviewMap.tsx

```diff
@@ -34,6 +36,7 @@ const SocialLocationPreview = (props: {
   return (
     <Wrapper border={props.border}>
       <MapHeaderHeader>US COVID Risk &amp; Vaccine Tracker</MapHeaderHeader>
+      <LastUpdatedWrapper>Updated {lastUpdatedDateString}</LastUpdatedWrapper>
```

Comment:
Note this changes the displayed date from the current date when the image was generated to the date the model was updated, which I believe to be the correct value to show.  If I have that wrong, LMK.

Reply:
per Igor in ticket, "match the homepage", which this does.